test/modules/returns/controllers/edit.js

```diff
@@ -694,6 +696,50 @@ experiment('edit controller', () => {
     });
   });
 
+  experiment('getMeterReset', () => {
+    test('it should take you to water/returns/internal/form page with returns data', async () => {
```

Comment:
I misread this thinking the route was doing a redirect, could we maybe say
`it should render...`